Physical_Engine/Create/Wall.cs

```diff
@@ -87,62 +72,58 @@ public static Wall Wall(IConstruction construction, ICurve bottomEdge, double he
         [Output("Wall", "The created physical Wall.")]
         public static Wall Wall(Line line, double height, IConstruction construction, Offset offset = Offset.Undefined, string name = "")
         {
-            if(line == null)
+            if (line == null)
             {
                 BH.Engine.Base.Compute.RecordError("Cannot create a Physical.Wall from a null line.");
                 return null;
             }
 
             Polyline boundary = new Polyline();
-
             Vector move = Vector.ZAxis * height;
-
             boundary.ControlPoints.Add(line.Start);
             boundary.ControlPoints.Add(line.End);
             boundary.ControlPoints.Add(line.End + move);
             boundary.ControlPoints.Add(line.Start + move);
             boundary.ControlPoints.Add(line.Start);
-
-            return new Wall
-            {
-                Location = Geometry.Create.PlanarSurface(boundary),
-                Construction = construction,
-                Offset = offset,
-                Name = name
-            };
+            return new Wall{Location = Geometry.Create.PlanarSurface(boundary), Construction = construction, Offset = offset, Name = name};
         }
 
         /***************************************************/
-
         [Description("Creates physical wall object.")]
         [Input("construction", "Construction of the wall.")]
         [Input("edges", "External edges of the wall (Profile - planar closed curve).")]
         [Input("internalEdges", "Internal edges of wall (profile).")]
         [Output("wall", "A physical wall.")]
-        public static Wall Wall(IConstruction construction, ICurve edges, IEnumerable<ICurve> internalEdges)
+        public static Wall Wall(IConstruction construction, ICurve edges, IEnumerable<ICurve> internalEdges = null)
```

Comment:
why do you default `internalEdges` to `null` for walls but not for floors?

Reply:
good catch, both should default to null.